src/sass_types/sass_value_wrapper.h

```diff
@@ -88,6 +92,65 @@ namespace SassTypes
   }
 
   template <class T>
+  NAN_METHOD(SassValueWrapper<T>::Operation) {
+    if (info.Length() < 2) {
+      return Nan::ThrowTypeError("Not enough arguments for operation");
+    }
+    if (!info[0]->IsString()) {
+      return Nan::ThrowTypeError("Operation value should be a string");
```

Comment:
Tried to stay as close as possible to other error messages.
But I just couldn't help wondering, why `should` and not `must`?
To me it looks like a very strict constraint one `should` meet :wink: